platform/7D2.112/consts.h

```diff
@@ -58,7 +59,7 @@
 
 #define CANON_SHUTTER_RATING 200000
 
-    #define DISPLAY_IS_ON               0x1  // TODO: find real value
+#define DISPLAY_IS_ON ((MEM(0x027e98) & 0x100) ? 0 : 1)  // found at 0xfe0b6824 : MEM(0x027e98) == 0x10001(on) 0x10101(off)
```

Comment:
This should probably be:
(!(char)(MEM(0x27e99))) ?

See fe0b65ee(int display_state), which is called with either 0 or 1.

Reply:
Ok, done and tested